frontend/src/reports/TwoVariableReport.tsx

```diff
@@ -115,10 +115,10 @@ function TwoVariableReport(props: {
 
       const demo: BreakdownVar = getParameter(DEMOGRAPHIC_PARAM, RACE);
       setVariableConfig1(
-        demoParam1 ? demoParam1 : METRIC_CONFIG[props.dropdownVarId1][0]
+        demoParam1 ?? METRIC_CONFIG?.[props.dropdownVarId1]?.[0]
       );
       setVariableConfig2(
-        demoParam2 ? demoParam2 : METRIC_CONFIG[props.dropdownVarId2][0]
+        demoParam2 ?? METRIC_CONFIG?.[props.dropdownVarId2]?.[0]
```

Comment:
ya ive been trying to default to using `??` instead of `||` for these short circuit assignments unless i specifically want to fail nullish stuff like `""` and `0`